services/rfq/relayer/quoter/quoter.go

```diff
@@ -364,13 +364,19 @@ func (m *Manager) generateActiveRFQ(ctx context.Context, msg *model.ActiveRFQMes
 	}
 	span.SetAttributes(attribute.String("request_id", rfqRequest.RequestID))
 
+	depositAmount, ok := new(big.Int).SetString(rfqRequest.Data.OriginAmount, 10)
```

Comment:
Could we keep this as originAmount instead of depositAmount for consistency?

To be clear I don't really have preference between originAmount and depositAmount but I do think it's cleaner to keep the naming consistent across the codebase. If you do prefer depositAmount we can run with that, but we should change it everywhere

Reply:
yeah youre right - actually for even more clarity i'd like to go with "originAmountExact" which is also consistent with swap interfaces